libexec/tfenv-resolve-version

```diff
@@ -142,6 +142,28 @@ if [[ "${version_requested}" =~ ^min-required$ ]]; then
   version_requested="${min_required}";
 fi;
 
+if [[ "${version_requested}" =~ ^latest-allowed$ ]]; then
```

Comment:
Does this case not need to be accounted for in tfenv-version-name and tfenv-uninstall also?

Reply:
Thanks